napari/utils/_testsupport.py

```diff
@@ -286,7 +300,7 @@ def actual_factory(
             # in particular with VisPyCanvas, it looks like if a traceback keeps
             # contains the type, then instances are still attached to the type.
             # I'm not too sure why this is the case though.
-            if _strict == 'raise':
+            if _strict:
```

Comment:
@Czaki could you explain the reason for this change?

I am doing some clean up of typos here and see that the docstring for `strict_qt` still mentions 'raise'? I don't think there is a way to raise just a warning any more. Just wanted to double check this before I update the docstring to new behaviour. Thanks

Reply:
I do not remember. Maybe this change should be reverted.

Reply:
You would know better than me! Could it be left over from when you were testing/working on the PR?